spec/models/proposal_spec.rb

```diff
@@ -952,6 +952,18 @@
       expect(result).to eq [proposal2]
     end
 
+    it "should not return archived proposals" do
+      archived_proposal = create(:proposal, :archived)
+      result = Proposal.recommendations(user)
+      expect(result.size).to eq(0)
```

Comment:
I think the test scenario would be more robust if there was a non-archived proposal so we know the filter is working (letting through only the ones that should). Same at the other spec at line 961